qi-lib/flow/core/deforest-syntax.rkt

```diff
@@ -1,16 +1,16 @@
 #lang racket/base
 
-(provide fusable-stream-producer
```

Comment:
This (3de37116fbdd68f27244e2cb86d5914131f45017) mostly looks like a fixup of the previous commit; if so, can it be squashed
into that one?